pom.xml

```diff
@@ -280,6 +280,13 @@ under the License.
                 <scope>test</scope>
             </dependency>
 
+            <!-- For dependency convergence -->
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-compress</artifactId>
+                <version>1.23.0</version>
+            </dependency>
+
```

Comment:
I guess we do not need it anymore since it is already included into pom
lines 297-302

Reply:
done